### ISSUE TITLE
Installation / Documentation updates

### ISSUE THREAD

**Issue by collincusce** [transmute-industries/transmute-framework#97]
Great work guys!

Installation needs clarity, though. 

Node v6.11.1 or higher is in package.json, could spell that out in docs.

Test RPC is fine, right? So I run `yarn truffle:migrate` with testrpc up and it should be aces, ya?

Just ran `yarn truffle:test` and got 

> yarn run v1.2.1
> $ truffle test
> Using network 'development'.
> 
> /home/ccusce/dev/arcticpup/proj1/test/Security/RBAC.spec.js:41
>         before(async () => {
>                      ^
> 
> SyntaxError: Unexpected token (
>     at createScript (vm.js:56:10)
>     at Object.runInThisContext (vm.js:97:10)
>     at Module._compile (module.js:542:28)
>     at Object.Module._extensions..js (module.js:579:10)
>     at Module.load (module.js:487:32)
>     at tryModuleLoad (module.js:446:12)
>     at Function.Module._load (module.js:438:3)
>     at Module.require (module.js:497:17)
>     at require (internal/module.js:20:19)
>     at /home/ccusce/dev/arcticpup/proj1/node_modules/mocha/lib/mocha.js:231:27
> error Command failed with exit code 1.
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

This means that I actually need node v8.7.0 I think? Fixed the error, looked it up, async is supported then. 6.xx doesn't support async and the libraries I can install do not support the way you're using it syntactically. 

I get new truffle:test errors... https://gist.github.com/collincusce/da85c4ba5afc52fbc175a5f48e86dfa3

Tried generating docs, get errors, etc etc.

So basically, I think some installation clarity would help out immensely. 

--Collin
